integration_tests/index/test_index_data.py

```diff
@@ -284,18 +217,18 @@ def test_transactions_api_ctx_mgr(index,
     with pytest.raises(Exception) as e:
         with index.transaction() as trans:
             assert index.datasets.get(ds1.id) is None
-            index.datasets.add(ds1)
+            index.datasets.add(ds1, False)
```

Comment:
I wasn't sure what this `False` was for, and had to go and look up that it's `with_lineage=False`.

I think it should be a keyword argument, and I even think the method should change that argument to be [keyword only](https://peps.python.org/pep-3102/).

Reply:
Added keyword to test calls.